client/src/App.js

```diff
@@ -23,9 +23,9 @@ function App() {
                 <BrowserRouter>
                     <Switch>
                         <Route exact path="/dashboard" component={Dashboard} />
-                        <ProtectedRoute path="/forum" component={Forum} />
+                        <Route path="/forum" component={Forum} />
                         <ProtectedRoute path="/groups" component={Groups} />
-                        <ProtectedRoute path="/chat" component={Chat} />
```

Comment:
Are we switching out of Protected Routes?

Reply:
This branch is fe-refactor-sidebar-#31. 
Yes, those routes are protected routes.

Reply:
Ahh okay. It said #46 in the PR. Just wanted to make sure I had the correct one.